### PR TITLE
530 show/hide create nav link

### LINKED ISSUE #530: Don't show create tab in navigation
### User story
Don't show create tab in navigation unless a user has a new booking in the process of being created.

### PR DESCRIPTION
**Pull request by KeeganWoods1**
##### _What existing problem does the pull request solve?_

Nav bar will only show the "create" nav-item while a booking request form is currently in the works; otherwise the "create" nav-item will be hidden.

##### _List all issues this PR resolves_
- closes #530 

##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### Sub-tasks Checklist
- [ ] UI Implementation
- [ ] UI Test
- [ ] Backend Implementation
- [ ] Backend Tests
- [ ] Integration Tests
- [ ] System Tests
- [ ] Documentation

### CODE REVIEW

**Review by EvanDime — COMMENTED** [review]
@a-drew Do you know if there is a way to check the local session for the booking info and use that in the vue to check for the create button?

File: resources/js/Pages/Admin/BookingRequests/CreateBookingRequestModal.vue [hunk 1]
```diff
@@ -199,6 +203,9 @@ export default {
         this.createBookingRequestForm.reference.push(file);
       }
     },
+    setCreate(val) {
+      localStorage.create = val;
+    },
```
Comment (alexstojda) [inline resources/js/Pages/Admin/BookingRequests/CreateBookingRequestModal.vue:208]:
```suggestion
    setLocalIsCreating(val) {
      localStorage.isCreatingBooking = val;
    },
```
the `setCreate` function name is a bit ambiguous, could you change it to be a bit more descriptive, as well as the `create` property. Included a suggestion for naming

**Review by alexstojda — CHANGES_REQUESTED** [review]
one comment!

File: resources/js/Pages/Admin/BookingRequests/CreateBookingRequestModal.vue (hunk 1 above)
Comment (KeeganWoods1) [reply resources/js/Pages/Admin/BookingRequests/CreateBookingRequestModal.vue]: Agreed, and done :)

**Comment by alexstojda** [comment]
> @a-drew Do you know if there is a way to check the local session for the booking info and use that in the vue to check for the create button?
> 
> if not, this manual bool setting is fine for the frontend

@EvanDime Session is stored server-side :(

**Review by alexstojda — DISMISSED** [review]

**Comment by EvanDime** [comment]
> > @a-drew Do you know if there is a way to check the local session for the booking info and use that in the vue to check for the create button?
> > if not, this manual bool setting is fine for the frontend
> 
> @EvanDime Session is stored server-side :(

I figured. Was hopping maybe there would be a way. Then I approve

**Review by EvanDime — DISMISSED** [review]

**Review by EvanDime — APPROVED** [review]

**Review by alexstojda — APPROVED** [review]

**Pull request closed without merging**